src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
```

Comment:
The imported 'Movie' type is not being used in this file. Consider removing the import statement if it's not needed.